src/components/Filter/Filter.css

```
@@ -0,0 +1,27 @@
+.filter {
+    border-bottom: 0.06rem solid #555;
+    margin-bottom: 1.25rem;
+    padding-bottom: 1.25rem;
+}
+
+.filter-options {
+    display: flex;
+    justify-content: space-between;
+    gap: 3rem;
+}
+
+.filter-options div {
+    flex: 1 1 0;
+}
```

Nesse código vc tem uma classe css declarada, e logo em seguida declara ela novamente para adicionar estilo a divs. Essas instruções podem estar juntas em um único bloco
```suggestion
.filter-options {
    display: flex;
    justify-content: space-between;
    gap: 3rem;
    div {
        flex: 1 1 0;
    }
}
```